guide/src/reference/attributes/on-js-imports/indexing-getter-setter-deleter.md

```diff
@@ -0,0 +1,57 @@
+# `indexing_getter`, `indexing_setter`, and `indexing_deleter`
+
+These three attributes indicate that a method is a getter, setter, or deleter
+method on the receiver object itself, rather than one of the receiver's
+properties. This is useful for binding to types that use `Proxy` to intercept
+property accesses.
```

Comment:
> These three attributes indicate that a method is a getter, setter, or deleter method on the receiver object itself, rather than one of the receiver's properties.

This is still a bit confusing (and looks like it is not changed yet comparing to previous version).
I read this as, for example for setter, either "`object = value`, rather than `object.property = value`" or "`object.property = value`, rather than `object.property.anotherProperty = value`", both senses are wrong. The actual difference is in whether access is static or dynamic, otherwise they are pretty similar.

It is not only used for `Proxy`, that is just an example that can be implemented in JS, but its main usage is for built-in types in browsers. So something like this would probably be better:
> This is useful for binding to types that intercept property accesses (for example, some build-in types in browsers or objects that are wrapped with `Proxy`).